n2y/cache.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import sqlite3
+import pickle
+
+from os.path import exists
+
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+class Cache:
+    """Cache Notion block entities in local storage"""
+
+    def __init__(self, client):
+        
+        self.client = client
+        self.cache_file = ".n2ycache"
```

Comment:
We'll probably want to pass the full path of the cache file into the __init__ as an argument. Also, we'll want to create the file if it doesn't exist and initialize the SQL tables.